deploy/codebuild-deploy.sh

```diff
@@ -15,4 +15,4 @@ export SERVICE_NAME="${BASH_REMATCH[2]}"
 #
 # The "*" in the scope is there to handle that package names are prefixed with
 # their directory so we can filter them with Lerna.
-npx lerna run --npm-client npm --stream --scope "*.$SERVICE_NAME" codebuild-deploy
+npx lerna@5.5.1 run --npm-client=npm --stream --loglevel=verbose --scope "*.$SERVICE_NAME" codebuild-deploy || pwd && ls -la ../ && cat ../lerna-debug.log
```

Comment:
This could just be:
`npx lerna@5.5.1 run --npm-client=npm --stream --scope "*.$SERVICE_NAME" codebuild-deploy`
The other bits were just to help debug